docs/UsersGuide.asciidoc

```diff
@@ -298,29 +305,89 @@ openqa-client isos post ISO=my_iso.iso DISTRI=my_distri FLAVOR=sweet \
 
 ==== Asset handling ====
 
-Multiple parameters exist to reference assets to be used by tests, for example
-`ISO`, `HDD`, `ASSET`, `KERNEL`, `INITRD`. Corresponding files must be
-provided within the path +/var/lib/openqa/share/factory+ local to the openQA
-web-UI. All assets specified in this way in any job are tracked by openQA and
-considered by the automatic cleanup described under 'Asset cleanup'.
-
-Multiple options can be used based on special suffix types. These types can
-also be combined, for example ISO_1_DECOMPRESS_URL.
+Multiple parameters exist to reference "assets" to be used by tests. "Assets" are essentially
+content that is stored by the openQA web-UI and provided to the workers; when sending jobs to
+os-autoinst on the workers, openQA adjusts the parameter values to refer to an absolute path
+where the worker will be able to access the content. Some assets can also be produced by a job,
+sent back to the web-UI, and used by a later job. Assets can also be seen in the web-UI and
```

Comment:
i.e. they can be "published"

Reply:
i was trying to make this paragraph a kind of 'plain language' summary of the things that are explained in detail below. lemme see if i can work it in.